corehq/apps/geospatial/tests/test_views.py

```diff
@@ -453,6 +453,42 @@ def test_save_polygon(self):
             del feature['id']
         self.assertEqual(saved_polygons[0].geo_json, geo_json_data)
 
+    def _validate_error_message(self, response, message):
```

Comment:
nit: maybe rename to `_assert_error_message` ?

Reply:
That sounds good, renamed in [114b4d9](https://github.com/dimagi/commcare-hq/pull/34849/commits/114b4d9cf89e2867788bd3bb527c96045694aec2)